<robot name="edumip_robot">
  <link name="edumip_body">
    <visual>
      <origin xyz = "0 0 0.034"/>
      <geometry>
        <box size="0.05 0.035 0.1"/>
      </geometry>
      <material name="body_color">
        <color rgba="0 1 0 .8"/>
      </material>
    </visual>
  </link>

  <link name="wheelL">
    <visual>
     <origin rpy="0 1.57 0"/>
      <geometry>
        <cylinder length="0.01" radius="0.034"/>
      </geometry>
      <material name="wheel_color">
        <color rgba="1 0 0 .8"/>
      </material>
    </visual>
  </link>

  <link name="wheelR">
    <visual>
     <origin rpy="0 1.57 0"/>
      <geometry>
        <cylinder length="0.01" radius="0.034"/>
      </geometry>
      <material name="wheel_color">
        <color rgba="1 0 0 .8"/>
      </material>
    </visual>
  </link>

  <joint name="jointL" type="continuous">
    <origin xyz="0.035 0 0" rpy="0 0 0"/>
    <parent link="edumip_body"/>
    <child link="wheelL"/>
  </joint>

  <joint name="jointR" type="continuous">
    <origin xyz="-0.035 0 0" rpy="0 0 0"/>
    <parent link="edumip_body"/>
    <child link="wheelR"/>
  </joint>

</robot>